EFCore/MAUI/Readme.md

```diff
@@ -0,0 +1,876 @@
+         editorUser.Roles.Add(defaultRole);
+
+         //create Viewer User/Role
+         var viewerUser = ObjectSpace.CreateObject<ApplicationUser>();
+         viewerUser.UserName = "Viewer";
+         var viewerRole = ObjectSpace.CreateObject<PermissionPolicyRole>();
+         viewerRole.Name = "ViewerRole";
+         viewerRole.AddTypePermission<Post>(SecurityOperations.Read,  SecurityPermissionState.Allow);
+         viewerRole.AddTypePermission<ApplicationUser>(SecurityOperations.Read,  SecurityPermissionState.Allow);
+         viewerUser.Roles.Add(viewerRole);
+         viewerUser.Roles.Add(defaultRole);
+ 
+         //commit
+         ObjectSpace.CommitChanges();
+ 
+         //assign authentication type
+         foreach (var user in new[] { editorUser, viewerUser }. Cast<ISecurityUserWithLoginInfo>()) {
+             user.CreateUserLoginInfo(SecurityDefaults.PasswordAuthentication,
+                 ObjectSpace.GetKeyValueAsString(user));
+         }
+ 
+         //sample posts
+         var post = ObjectSpace.CreateObject<Post>();
+         post.Title = "Hello World";
+         post.Content = "This is a FREE API for everybody";
+         post.Author=editorUser;
+         post = ObjectSpace.CreateObject<Post>();
+         post.Title = "Hello MAUI";
+         post.Content = "Please smash the like button to help our videos get  discovered";
+         post.Author=editorUser;
+     }
+   ```
+
+### Use Swagger UI to test the Web API service
+
+At this point, you can already run your Web API service and use the Swagger interface to authenticate as one of the previously defined users and test the generated endpoints (for example, query the available posts). See the following article for additional information: [Test the Web API with Swagger or Postman](https://docs.devexpress.com/eXpressAppFramework/404281/backend-web-api-service/test-the-web-api-with-swagger-postman).
+
+![](../../images/MAUI/SwaggerQueryPosts.png)
+
+![](../../images/MAUI/SwaggerPostsResponse.png)
+
+## Build and test a basic .NET MAUI app (Frontend)
+
+### Create a .NET MAUI project
+
+> **NOTE** 
+>
+> Debugging configurations for both iOS and Android can be complex, so it is not feasible to provide all possible scenarios. We tested the app on Windows 10 with Visual Studio 2022. For iOS tests, we paired a remote Mac. For Android tests, we utilized the built-in emulator.
+
+1. Open the the _Create a new project_ window in Visual Studio. Select the DevExpress .NET MAUI option and start the wizard. 
+
+  ![](../../images/MAUI/MAUINewProjectWizard.png)
+
+2. Choose both the _iOS & Android_ platforms, the _Tabbed_ layout, and the following controls: _Collection View_, _Data Editors_, and _Data Forms_.
+
+  ![](../../images/MAUI/MAUINewProjectWizardConfig.png)
+
+### Modify platform specific settings
+   
+1. In the `Platform/Android/AndroidManifest.xml` file, to set up the file provider, which is required for the application to store downloaded reports:
```

Comment:
I'd like to add a phrase that introduces the code below if possible. Does the reader need to add the code below to the `Platform/Android/AndroidManifest.xml` file? For example, do you mean:

`1. In the `Platform/Android/AndroidManifest.xml` file, add the code below to set up the file provider, which is required for the application to store downloaded reports:`

Reply:
Thank you